app/models/server/raw/LivechatRooms.js

```diff
@@ -145,15 +145,15 @@ export class LivechatRoomsRaw extends BaseRaw {
 			});
 		}
 		const params = [...firstParams, group, presentationProject];
+		if (options.sort) {
```

Comment:
This won't work.
If there is no `options.sort` defined, the `default sort` won't be assigned.

To clarify, you don't need the `if` here.